clang/lib/Parse/ParseExpr.cpp

```diff
@@ -2237,6 +2237,9 @@ Parser::ParsePostfixExpressionSuffix(ExprResult LHS) {
             if (PP.isCodeCompletionReached() && !CalledSignatureHelp)
               RunSignatureHelp();
             LHS = ExprError();
+          } else if (!HasError && HasTrailingComma) {
+            // FIXME: add a FIXIT to remove the trailing comma.
+            Diag(Tok, diag::err_extraneous_trailing_comma);
```

Comment:
I think we should just reuse `err_expected_expression`. The important point here is that there is a diagnostic.
Note sure this happens often enough to warrant a fixit

Reply:
Reused the existing err_expected_expression.

I initially thought that the new `err_extraneous_trailing_comma` provided a clearer message than `err_expected_expression`, but it only applies to a specific test case (the one added in this PR).

For other cases, such as `function2Arg(1, );`, the new diagnostic is incorrect.